src/libs/actions/PaymentMethods.js

```diff
@@ -95,8 +123,42 @@ function clearDebitCardFormErrorAndSubmit() {
     });
 }
 
+/**
+ * Set the necessary data for wallet transfer
+ * @param {Number} currentBalance
+ * @param {Number} selectedAccountID
+ */
+function saveWalletTransferAmountAndAccount(currentBalance, selectedAccountID) {
+    Onyx.set(ONYXKEYS.WALLET_TRANSFER, {
+        transferAmount: currentBalance - CONST.WALLET.TRANSFER_BALANCE_FEE,
+        selectedAccountID,
+        filterPaymentMethodType: null,
+        loading: false,
+        completed: false,
+    });
+}
+
+/**
+ * Update selected accountID and other data for wallet transfer
+ * @param {Object} data
+ */
+function updateWalletTransferData(data) {
+    Onyx.merge(ONYXKEYS.WALLET_TRANSFER, data);
+}
+
+/**
+ * Cancel the wallet transfer
+ */
+function cancelWalletTransfer() {
```

Comment:
I think this would be clearer if we added "pending wallet transfer request" to all of these method names? I'm having trouble telling the difference between the action that calls the API and the one that "prepares" the data we need to make a request.

Reply:
`cancelWalletTransfer()` sounds like an action that will call the API and cancel something.